docs/linking.md

```diff
@@ -106,13 +106,13 @@ If your app is using [Universal Links](https://developer.apple.com/library/prere
 
 There are two ways to handle URLs that open your app.
 
-#### 1. If the app is already open, the app is foregrounded and a Linking event is fired
+#### 1. If the app is already open, the app is foregrounded and a Linking 'url' event is fired
 
-You can handle these events with `Linking.addEventListener(url, callback)`.
+You can handle these events with `Linking.addEventListener('url', callback)` -- it calls `callback({ url })` with the linked URL
```

Comment:
From the code it looks like the callback returns just the `url` string not wrapped with the object as added comment suggest: 
* https://github.com/facebook/react-native/blob/master/RNTester/js/RNTesterApp.ios.js#L163
* https://github.com/facebook/react-native/blob/master/RNTester/js/utils/URIActionMap.js#L35

Reply:
Are you sure RNTester is being used and tested against?

I have working code that looks like this:
```
Linking.addEventListener('url', ({ url }) => {
  this.props.updateDeepLink(url);
});
```

Reply:
Here's a more terse example from working code, RN 0.61.2:
```js
Linking.addEventListener('url', event => {
  console.log('event', Object.keys(event));
});
// outputs:
// LOG  event ["url"]
```

Reply:
Thank you for checking on that! I was able to confirm using latest version of RN that indeed Linking listener returns an event with only one key which is `url`. The RNTester is used and updated, but as this case shows, not every thing is 100% spot on. The work on the RNTesters lately is a bit on hold due to refactor project from MLH team.